docs/en/observability/apm/data-streams.asciidoc

```diff
@@ -109,7 +109,7 @@ For example, consider traces that would originally be indexed to `traces-apm-def
 ]
 ----
 
-For more about other ingest pipelines called by default, e.g. `traces-apm@custom`, see {fleet-guide}/data-streams.html#data-streams-pipelines[integration data streams ingest pipelines].
+For more about other ingest pipelines called by default, e.g. `traces-apm@custom`, see definition of `\*-apm*@default-pipeline` ingest pipelines from {es} apm-data plugin.
```

Comment:
Should this link somewhere? Usually when we say "see {something}" the {something} is a linked resource.

Reply:
There are no docs around it, not like integration data streams ingest pipelines. What I was trying to convey was that users should just go to ES and check out the relevant pipelines themselves. I don't feel comfortable linking to the [actual source code](https://github.com/elastic/elasticsearch/tree/main/x-pack/plugin/apm-data), as pinning source code version to doc version is not feasible. Any good idea how to rewrite this?

Reply:
>go to ES and check out the relevant pipelines themselves

Ok. This makes sense to me. Where would they go or what would they do to find the relevant pipelines? Stack management in Kibana? Using an API? Something else?

Reply:
I'm thinking about stack management, but if a power user likes API, they can do so too.

Reply:
>I'm thinking about stack management, but if a power user likes API, they can do so too.

We can keep it simple with just Stack management.

What about something like this? Feel free to adjust as needed.

```suggestion
To find other ingest pipelines from the {es} apm-data plugin that are called by default, go to {kibana-ref}/management.html[*Stack management*] → *Ingest pipelines* and search for `apm`.
Default APM ingest pipelines will follow the pattern `*-apm*@default-pipeline`.
```

Reply:
Looks perfect, thanks!